### PR TITLE
Portfolio: details popup window

### PR DESCRIPTION
**Pull request by a-laarabi**
# What?
Creatin the popup window to see more details about any project in the work section

### CODE REVIEW

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Invalid Code Review Request ❌

Hi @a-laarabi, This is @sja-thedude your code reviewer.

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

You have submitted a project, that doesn't have commits from all the members in your pair programming group. Look below the submission guidelines:
Screenshot 2022-06-16 at 7 04 16 AM

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: index.html [hunk 1]
```diff
@@ -53,92 +53,33 @@ <h2>LET’S CONNECT</h2>
...
+                        <li><a href="#">html</a></li>
+                        <li><a href="#">css</a></li>
+                        <li><a href="#">javaScript</a></li>
+                    </ul>
+                    <button type="button" class="see button1">See live <img src="img/Icon-live.png" alt="live demo"></button>
+                    <button type="button" class="see button2">See Source <img src="img/Icon-button-GitHub.png" alt="see source"></button>
+                </div>
+            </div>
+        </div>
+    </section>
```
Comment (keroloussamy) [inline index.html:79]: - [ ] Kindly on your popup window you should display the right data for the project, currently, you are displaying the same data for all projects.

File: style.css [hunk 2]
```diff
@@ -563,6 +589,55 @@ textarea {
...
+  width: 80%;
+  min-height: 90vh;
+  border-radius: 18px;
+  border: 1px solid #ebecf0;
+  padding: 16px;
+  background-color: #fff;
+  margin-top: 60%;
+  padding-bottom: 20px;
+  margin-bottom: 20px;
+}
```
Comment (keroloussamy) [inline style.css:622]: - [ ] Kindly I can't see the close button and the name of the project on your mobile version, kindly fix them to match the Figima design.

Figima design
image

Your design
image

**Review by keroloussamy — CHANGES_REQUESTED** [review]
Hi @a-laarabi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️

## Required Changes ♻️
_Check the comments under the review._




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**